.ci/update_version.sh

```diff
@@ -10,28 +10,31 @@ function version_gt() {
 }
 
 
-if [[ $# -ne 1 ]]; then
-  echo "No arguments supplied. Please provide the release version" 1>&2
+if [[ $# < 1 || $# > 2 ]]; then
+  echo "Incorrect number of arguments. Usage: ${0%*/} <version> [<docker-tag>]" 1>&2
   echo "Terminating the script execution." 1>&2
   exit 1
 fi
 
 release_version="$1"
+docker_image_tag="$2"
```

Comment:
Can you post an example of what gets passed to this script?

Reply:
Second arg can be empty. Above checks for 1 or 2 arguments. If only one is provided, $2 will be empty string.

Reply:
Sample usage:

.ci/update_version.sh 2.3.3.0 2.3.3.0-b22

Reply:
If we provide only one arg, we get old behavior of searching